content/04.results.md

```diff
@@ -125,41 +126,37 @@ We observed 8% (16/195) of tumors to be _C11orf95::RELA_ fusion positive ependym
+B, Co-occurrence and mutual exclusivity of nonsynonymous mutations between genes;
+The co-occurrence score is defined as $I(-\log_{10}(P))$ where $P$ is defined by Fisher's exact test and $I$ is 1 when mutations co-occur more often than expected and -1 when exclusivity is more common;
+C, _De novo_ CNS signatures;
+D, The number of SV breaks significantly correlate with CNV breaks (Adjusted R = 0.436, p = 1.08e-37). 
+E, Chromothripsis frequency across pediatric brain tumors shown by `cancer_group` with N >= 3.
+](images/main/Figure3.png){#fig:Fig3}
 
-![This figure shows dominant statuses for small copy number consensus segments (< 10 Mb) across the genome, where each square represents a ~1 Mb binned section of the genome.
-A dominant status is declared if one status is >75% coverage.
-Unstable indicates multiple non-neutral statuses totaling coverage >75%.
-Copy number segments longer than 10 Mb have been removed from the figure for easier interpretability.](https://github.com/AlexsLemonade/OpenPBTA-analysis/raw/master/figures/pngs/cn_status_heatmap.png){#fig:copy-number-status-heatmap width="7in"}
 
-### Recurrence and co-occurrence of mutations
+### Genomic instability of pediatric brain tumors 
 
-This section will discuss the genes and regions that are repeatedly mutated within and between cancer types.
-The occurrence of mutations in affecting particular genes, separated by tumor type is shown in Figure {@fig:occurrence-plots}A, with significant co-occurrence across all types illustrated in Figure {@fig:occurrence-plots}B.
+We developed a method for generating consensus copy number altered regions from MantaSV, CNVkit, and ControlFREEC (**STAR Methods**). 
+This resulted in high-confidence gains, losses, amplifications, and deep deletion calls used as input for breakpoint density (**Table S2**) and chromothripsis analysis.
+We observed that HGATs, followed by medulloblastomas, have the most unstable genomes (**Figure S3A**). By contrast, craniopharyngiomas generally lack somatic copy number variation. These patterns of copy number variation largely align with estimates of tumor mutational burden (**Figure S2G**). 
+The number of SV and CNV breakpoints were significantly correlated across tumors (p = 1.08e-37) (**Figure {@fig:Fig3}D**) and as expected, the number of chromothripsis regions called increases as breakpoint density increases (**Figure S3B-C**).
+Chromothripsis events were observed in 41% (N = 19/46) of non-midline high-grade gliomas and 28.2% (N = 11/39) of DMGs (**Figure {@fig:Fig3}E**). 
+We also found evidence of chromothripsis in over 15% of ATRTs, medulloblastomas, ependymomas, meningiomas, germinomas, CNS embryonal tumors, glial-neuronal tumors, chordomas, ETMRs, metastatic secondary tumors, and sarcomas to have evidence of chromothripsis, highlighting the genomic instability and complexity of pediatric brain tumors.
```

Comment:
I think I commented earlier about the length of this list, but if we are to keep it, the following edit is needed: 
```suggestion
We also found evidence of chromothripsis in over 15% of ATRTs, medulloblastomas, ependymomas, meningiomas, germinomas, CNS embryonal tumors, glial-neuronal tumors, chordomas, ETMRs, metastatic secondary tumors, and sarcomas, highlighting the genomic instability and complexity of pediatric brain tumors.
```

Reply:
>I think I commented earlier about the length of this list, but if we are to keep it, the following edit is needed:

I can reduce this a bit.